app/lib/submission_builder/formatting_methods.rb

```diff
@@ -26,6 +26,17 @@ def date_type(date)
       date.strftime("%F")
     end
 
+    def date_type_for_timezone(date)
+      return nil unless date.present?
+      date = case @submission.data_source.state_code
+             when 'ny', 'md', 'nc', 'nj'
+               date.in_time_zone('America/New_York')
+             else
+               date.in_time_zone('America/Phoenix')
+             end
```

Comment:
I discussed this with @tahsinaislam but capturing it here also: please create a `timezone` key in the StateInfomationService so that we don't need a case statement here and can instead use something like
```
timezone = StateFile::StateInformationService.timezone(@submission.data_source.state_code)
date.in_time_zone(timezone)